Sources/GoodReactor/Publishers/PassthroughPublisher.swift

```diff
@@ -1,13 +1,13 @@
 //
-//  Publisher.swift
+//  PassthroughPublisher.swift
 //  GoodReactor
 //
-//  Created by Filip Šašala on 28/08/2024.
+//  Created by Filip Šašala on 25/10/2024.
 //
 
 import Foundation
 
-public actor Publisher<Value: Sendable> {
+public actor PassthroughPublisher<Value: Sendable>: Publisher {
```

Comment:
toto ma od prveho pohladu trocha zmiatlo ze ci to nie je nativ :D ale tam to je subject tak asi ok 😄